File: Braintree/src/test/java/com/braintreepayments/api/internal/StreamHelperUnitTest.java
package com.braintreepayments.api.internal;

import org.junit.Test;
import org.junit.runner.RunWith;
import org.robolectric.RobolectricTestRunner;

import java.io.IOException;
import java.io.InputStream;

import static com.braintreepayments.testutils.FixturesHelper.streamFromString;
import static junit.framework.Assert.assertEquals;

@RunWith(RobolectricTestRunner.class)
public class StreamHelperUnitTest {

    @Test
    public void getString_readsAStringFromAStream() throws IOException {
        InputStream inputStream = streamFromString("Test string");

        assertEquals("Test string", StreamHelper.getString(inputStream));
    }
}

File: BraintreeCore/src/test/java/com/braintreepayments/api/StreamHelperUnitTest.kt
package com.braintreepayments.api

import com.braintreepayments.api.StreamHelper.getString
import org.junit.Assert.assertEquals
import org.robolectric.RobolectricTestRunner
import org.junit.Test
import org.junit.runner.RunWith
import java.io.IOException

@RunWith(RobolectricTestRunner::class)
class StreamHelperUnitTest {
    @Test
    @Throws(IOException::class)
    fun getString_readsAStringFromAStream() {
        val inputStream = FixturesHelper.streamFromString("Test string")
        assertEquals("Test string", getString(inputStream))
    }
}
